src/shared/flow-types.js

```diff
@@ -54,7 +54,7 @@ export type SingleImageryProjectType = {
     // FIXME: we should use constants here, somehow flow is not happy with them
     projectType: 1 | 2 | 4,
     progress: number,
-    state: number,
+    status: string,
```

Comment:
interesting. why do we have to change it now? didn't we deprecate the state attribute already when switching to v2?

Reply:
We did, but somehow this state attribute wasn't seen by flow until I explicitly checked for `status`, and then it complained, which prompted me to change it. Basically, this line was never really used, it seems...